index.html

```diff
@@ -1,49 +1,73 @@
-  <link rel="stylesheet" href="style.css">
-  <link rel="preconnect" href="https://fonts.googleapis.com">
-  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;900&display=swap" rel="stylesheet">
-  <title>My Portfolio</title>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      src="https://kit.fontawesome.com/f8112dc1f2.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;900&display=swap"
+      rel="stylesheet"
+    />
+    <!-- <script src="https://code.jquery.com/jquery-3.6.0.js"></script> -->
 
-  <!-- Global site tag (gtag.js) - Google Analytics -->
-  <script async src="https://www.googletagmanager.com/gtag/js?id=G-EP43G5K157"></script>
-  <script>
-    window.dataLayer = window.dataLayer || [];
-    function gtag(){dataLayer.push(arguments);}
-    gtag('js', new Date());
+    <title>My Portfolio</title>
 
-    gtag('config', 'G-EP43G5K157');
-  </script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-EP43G5K157"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        dataLayer.push(arguments);
+      }
+      gtag('js', new Date());
 
-</head>
-<body>
-    <header>
-      <a class="wrap logo" href="#">Kehinde Audu</a>
-      <nav>
-        <ul class="menus">
-          <li><a href="#">Portfolio</a></li>
-          <li><a href="#">About</a></li>
-          <li><a href="#">Contact</a></li>
+      gtag('config', 'G-EP43G5K157');
+    </script>
+  </head>
+  <body>
+    <header class="header">
+      <nav class="navbar d-flex">
+        <a class="wrap logo" href="#">Kehinde Audu</a>
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#portfolio" class="nav-link">Portfolio</a>
+            <i class="fas fa-chevron-right"></i>
+          </li>
+          <li class="nav-item">
+            <a href="#about" class="nav-link">About</a>
+            <i class="fas fa-chevron-right"></i>
+          </li>
+          <li class="nav-item">
+            <a href="#contact" class="nav-link">Contact</a>
+            <i class="fas fa-chevron-right"></i>
+          </li>
         </ul>
-        <i class="fas fa-bars"></i>
-        <i class="far fa-envelope"></i>
+        <div class="hamburger wrap">
+          <i class="fas fa-bars"></i>
+        </div>
+        <div class="close">&times;</div>
```

Comment:
- [ ] When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears. But for your page, the closing (X) element is still left on display, say after selecting the contact option from a mobile menu, on scrolling up, there is a closing element instead of the Hamburg icon.
![Screenshot from 2021-10-06 14-27-11](https://user-images.githubusercontent.com/65501535/136194193-256a55bc-5128-427b-80aa-862082cd5946.png)

Reply:
Hello mke2111, thanks so much for your feedback. Could you explain how I can achieve that please, I have tried but I couldn't get it. Please help if you can. Thanks

Reply:
This issue is well resolved! ✔️